.travis.yml

```diff
@@ -1,39 +1,66 @@
 language: go
+go: "1.10"
 
-go:
-  - "1.10"
-
```

Comment:
I think it's a good idea to test "1.x" in addition to 1.10 to know when something breaks in advance

Reply:
IIRC tests fail on 1.11

Reply:
Oh. Can you provide a link to the log of CI build, that succeeded with 1.10, but failed with 1.11? We ought to start figuring this out, I'd rather have CI report "failed" for some time, than mask the issue.